tests/tests_unit/test_meta.py

```diff
@@ -19,10 +27,19 @@ def keep(path):
         ]
         return all(skip not in str(path.as_posix()) for skip in skip_list)
 
-    all_filepaths = Path("cognite/client/").glob("**/*.py")
     err_msg = "File: '{}' is missing 'from __future__ import annotations' at line=0"
-
-    for filepath in filter(keep, all_filepaths):
+    for filepath in filter(keep, ALL_FILEPATHS):
         with filepath.open("r") as file:
             # We just read the first line from each file:
             assert file.readline() == "from __future__ import annotations\n", err_msg.format(filepath)
+
+
+@pytest.mark.parametrize("cls", [CogniteResource, CogniteResourceList])
+def test_ensure_all_tests_use_camel_case_except_dump(cls):
+    method = "to_pandas"
```

Comment:
This seems a bit excessive?

Reply:
I make a variable for (almost) anything that is needed more than once. It is used three times. 🤷